test/language/expressions/greater-than/bigint-and-symbol.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2018 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Relational comparison of BigInt and Symbol values
+esid: sec-abstract-relational-comparison
+features: [BigInt, Symbol]
+---*/
+
+assert.throws(TypeError, function() {
+  3n > Symbol("2");
+}, "ToNumeric(rhs) throws.");
+
+assert.throws(TypeError, function() {
+  Symbol("2") > 3n;
+}, "ToNumeric(rhs) throws.");
```

Comment:
Would you mind adding something to either of the comments that differentiates them? For example: `"ToNumeric(BigInt) on RHS throws."`

Reply:
That's really good.